### PR TITLE
Summary of 3d_object_detection added

### PR DESCRIPTION
**Pull request by SinghRamK**

### CODE REVIEW

File: summaries/3d_object_detection.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
+# Center-based 3D Object Detection and Tracking
+
```
Comment (HatsuHaru23) [inline summaries/3d_object_detection.md:2]: Mention authors and the conference

File: summaries/3d_object_detection.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+    <td>57.5</td>
+  </tr>
+  <tr>
+    <td></td>
+    <td>Center-based</td>
+    <td>66.5</td>
+    <td>57.4</td>
+    <td>62.0</td>
+  </tr>
+</table>
```
Comment (HatsuHaru23) [inline summaries/3d_object_detection.md:163]: Give two cents and resources(paper link at least)

File: summaries/3d_object_detection.md (hunk 1 above)
Comment (SinghRamK) [reply summaries/3d_object_detection.md:2]: can you see again i added the authors name and conference also two cents and resources. please check it once more

Comment (HatsuHaru23) [reply summaries/3d_object_detection.md:2]: Hi. Nice work. Just a small mistake, change CPVR to CVPR

**Pull request merged**